modules/obam/manifests/init.pp

```diff
@@ -1,5 +1,5 @@
 #----------------------------------------------------------------------------
-#  Copyright (c) 2019 WSO2, Inc. http://www.wso2.org
+#  Copyright (c) 2021 WSO2, Inc. http://www.wso2.org
```

Comment:
@nizzlo Do not change the copyright notice of the existing files. You need to add the copyright notice with the year the file is added to the repo.

Reply:
fixed them